lib/view/monthView.dart

```diff
@@ -56,96 +58,156 @@ class _MonthViewState extends State<MonthView> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: getTopBar(Event, "monthly", context, this),
-      body: Column(
-        children: [
-          TableCalendar(
-            firstDay: getMonthAsDateTime(DateTime.now()),
-            lastDay: getDateOnly(getNextMonthAsDateTime(DateTime.now()),
-                offsetDays: -1),
-            focusedDay: _focusedDay,
-            calendarFormat: _calendarFormat,
-            selectedDayPredicate: (day) {
-              return isSameDay(_selectedDay, day);
-            },
-            onDaySelected: (selectedDay, focusedDay) {
-              if (!isSameDay(_selectedDay, selectedDay)) {
-                setState(() {
-                  _selectedDay = selectedDay;
-                  _focusedDay = focusedDay;
-                });
+      appBar: AppBar(
+        elevation: 1,
+        shape: const RoundedRectangleBorder(
+            borderRadius: BorderRadius.only(
+          bottomLeft: Radius.circular(20),
+          bottomRight: Radius.circular(20),
+        )),
```

Comment:
Can you replace the appBar: with getTopBar functions I had put in for the daily weekly monthly views?

Reply:
sure